### PR TITLE
NAS-102196

### PR DESCRIPTION
**Pull request by dkmullen**
Adds dynamic labels to identify controllers in Interfaces, IPMI components

### CODE REVIEW

File: src/app/pages/network/ipmi/ipmi.component.ts [hunk 1]
```diff
@@ -16,9 +16,16 @@ import { T } from '../../../translate-marker';
...
+    diameter='25'
+    class="form-select-spinner"
+    id="ipmi_controller-spinner"
+    *ngIf="!currentControllerLabel">
+  </mat-spinner>
   <mat-select *ngIf="is_ha" #storageController name="controller" placeholder="Controller" (selectionChange)="loadData()" [(ngModel)]="remoteController">
-    <mat-option [value]="false">Current controller</mat-option>
-    <mat-option [value]="true">Failover controller</mat-option>
+
+    <mat-option [value]="false">Current Controller: TrueNAS Controller {{currentControllerLabel}}</mat-option>
```
Comment (wblock) [inline src/app/pages/network/ipmi/ipmi.component.ts]: This is pretty redundant.  "Current" and "Failover" are kind of ambiguous in meaning.  Let's just implement the correct terminology here:
```suggestion
    <mat-option [value]="false">Active: TrueNAS Controller {{currentControllerLabel}}</mat-option>
```

File: src/app/pages/network/ipmi/ipmi.component.ts [hunk 2]
```diff
@@ -16,9 +16,16 @@ import { T } from '../../../translate-marker';
...
+    <mat-option [value]="true">Failover Controller: TrueNAS Controller {{failoverControllerLabel}}</mat-option>
```
Comment (wblock) [inline src/app/pages/network/ipmi/ipmi.component.ts]:
```suggestion
    <mat-option [value]="true">Standby: TrueNAS Controller {{failoverControllerLabel}}</mat-option>
```

File: src/app/pages/network/ipmi/ipmi.component.ts [hunk 3]
```diff
@@ -132,6 +141,17 @@ export class IPMIComponent {
     if (window.localStorage.getItem('is_freenas') === 'false') {
       this.ws.call('failover.licensed').subscribe((is_ha) => {
         this.is_ha = is_ha;
+        if (this.is_ha) {
+          this.ws.call('failover.node').subscribe((node) => {
+            if (node === 'A') {
```
Comment (wblock) [inline src/app/pages/network/ipmi/ipmi.component.ts]: This whole block can be replaced with the ternary operator.  That assumes we will only ever have two controllers, but the if/else if block also does that.
```
this.currentControllerLabel = (node === 'A') ? '1' : '2';
this.failoverControllerLabel = (node === 'A') ? '2' : '1';
```

**Comment by lola27** [comment]
@dkmullen could you please resolve the conflicts? thanks

**Pull request merged**